nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -21,6 +21,17 @@ in {
         '';
       };
 
+      target = mkOption {
```

Comment:
Bikeshed: rename `target` to `operation`?

(`colmena` calls this arg `goal`, but that doesn't feel like a good fit when reading the `system.autoUpgrade.*` options IMHO.)

Reply:
Ok for me.